### PR TITLE
Add rake tasks for importing

### PR DESCRIPTION
**Pull request by amclelland**
Now users don't need to create their own! :tada:

### CODE REVIEW

**Comment by artfuldodger** [comment]
Need to update the README too :+1:

File: README.md [hunk 1]
```diff
@@ -42,26 +42,17 @@ Add your Pivotal Tracker API token to your secrets.yml:
 
 `tracker_api_token: <%= ENV["PIVOTAL_TRACKER_API_TOKEN"] %>`
 
-Create a new rake task called `import.rake`
+Initialize your database by running `rake import:pivotal_initial`
```
Comment (jakesutton) [inline README.md]: Fix dat namespace, yo.

Comment (jakesutton) [inline README.md]: Or just :hocho: this line?

File: lib/active_pivot/railtie.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'active_pivot'
+require 'rails'
+module ActivePivot
+  class Railtie < Rails::Railtie
+    rake_tasks do
+      load 'lib/tasks/import.rake'
```
Comment (jakesutton) [inline lib/active_pivot/railtie.rb:6]: Whee!

**Comment by jakesutton** [comment]
:cake:

**Pull request merged**